### DISCUSSION TITLE
New Options

### DISCUSSION THREAD

**Discussion by mirrorclone** [m4mallu/clonebot#5] [General]
A) first feature

Video
Audio
Document
Picture
Message(text)

B) two feature

Ability to manually delete tags like @channelname when copying messages

**Comment by m4mallu** [comment]
As u said, a custom tag should be there when copying @tagged files. This can be included in future updates. But when a tag in the file name cannot be changed as the file need to be renamed. so, tags included in captions can only be changed with this feature.
